src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -501,7 +509,8 @@ export class SkyAutocompleteComponent
             case 'up':
               this.searchResultsIndex--;
               if (this.searchResultsIndex < 0) {
-                this.searchResultsIndex = this.searchResults.length - 1;
+                // Fallback to 0 just in case results are async and aren't returned yet.
+                this.searchResultsIndex = Math.max(this.searchResults.length - 1, 0);
```

Comment:
☝️ Not a part of the scope of this PR, but was an easy fix. This prevents a console error if the end-user hits the up arrow while waiting on the search results to be returned.